src/ui/landing/components/landing.tsx

```diff
@@ -132,9 +131,6 @@ export class LandingComponent extends React.Component<Props, State> {
         justifyContent: 'center',
         alignItems: 'center',
         color: JolocomTheme.textStyles.subheadline.color
-      },
-      buttonContainer: {
```

Comment:
this edit creates an error in the component because the style of the Button still references this.styles.buttonContainer